AgentDiscoveries-Frontend/app/src/components/nav.jsx

```diff
@@ -32,6 +48,41 @@ export default class NavigationBar extends React.Component {
             isAdmin: isAdmin(),
             isAgent: isAgent()
         });
+        if( isLoggedIn() ) {
+            this.getProfilePicture();
```

Comment:
I'd be inclined to extract this bit into a `refreshUser` function or similar? That way we can call that in both the `componentWillMount` & here, and it's really clear what we're intending